### DISCUSSION TITLE
Richer text markup

### DISCUSSION THREAD

**Discussion by amjr108** [zim-desktop-wiki/zim-desktop-wiki#2179] [Ideas]
Zim is a great application, but sometimes its pages are just hard to read because of  minimal markup. For example, there is no backquotes for storing quotations. This idea can be extended to general fenced block with subtitle (quote, remark, inspiration, etc.)

Currently, I'm emulating these with codeblock of markdown type, but they're quite ugly.

**Comment by sojusnik** [comment]
Would love to see a new format type for quotations, f.i. by `>` at the beginning of a sentence, like done in GitHub.

**Comment by jaap-karssenberg** [comment]
I just use indent for quotations. That is also how block quotes are
typically rendered in html.

This would also be by far the easiest way to support quotation if/when we
want to support markdown in zim.

-- Jaap

**Reply by sojusnik** [reply]
IMHO `>` gives you more flexibility, because when using indent your text looks like normal text, whereas with a distinctive `>` formatting the text will stand out more.
At the moment I just use italic formatting for quotes, but it's very hackish.

**Comment by amjr108** [comment]
> That is also how block quotes are
> typically rendered in html.

And left border (bar) is quite common.

**Reply by sojusnik** [reply]
Exactly! Having this in Zim would be awesome!

**Comment by lorenrosen** [comment]
You can get a backquote by inserting that symbol, but that's not very convenient.
